install_config/install/planning.adoc

```diff
@@ -117,7 +117,7 @@ not supported.
 === Single Master and Multiple Nodes
 
 The following table describes an example environment for a single
-xref:../../architecture/infrastructure_components/kubernetes_infrastructure.adoc#master[master] (with embedded *etcd*)
+xref:../../architecture/infrastructure_components/kubernetes_infrastructure.adoc#master[master] (with *etcd* on the same host)
```

Comment:
> Embedded is not the same as merely being on the same host.
  
Right, but I thought that the change is more toward what we should be recommending.

> Note that it doesn't mention installing etcd. That's because it is using the etcd embedded in the openshift binary

Hmm OK. Maybe this is leading to something missing from the docs. i'll talk about this more in a comment below, but it's my understanding that we should not be recommending embedded etcd.